toolset/setup/linux/languages/mono.sh

```diff
@@ -16,8 +16,7 @@ echo "deb http://jenkins.mono-project.com/repo/debian sid main" | sudo tee /etc/
 sudo apt-get update
 
 # Find the most recent snapshot
-#SNAPSHOT=$(apt-cache search 'mono-snapshot-.*-assemblies' | cut -d'-' -f3 | tail -1)
-SNAPSHOT="20150202010831"
+SNAPSHOT=$(apt-cache search 'mono-snapshot-.*-assemblies' | cut -d'-' -f3 | tail -1)
```

Comment:
This line was purposefully changed to NOT find the most recent because the builds of mono would break day to day. We found the most recent (at the time) build that worked and used it directly. I am weary of returning to this code that finds the latest because of this same reason.